test/e2e/auth/audit.go

```diff
@@ -138,6 +139,26 @@ func expectAuditLines(f *framework.Framework, expected []auditEvent) {
 }
 
 func parseAuditLine(line string) (auditEvent, error) {
+	var e struct {
```

Comment:
Is this struct not already defined in API types we can reference?

Reply:
Updated to use the audit Event type.